### PR TITLE
#20 - February release notes

### LINKED ISSUE #20: February release notes
subj

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #20 
The list of changes is small this month, that's why I decided to describe docker permissions settings.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@g4s8 @genryxy please take a look, do we have smth to add?

**Comment by genryxy** [comment]
@olenagerasimova probably this [one](https://github.com/artipie/npm-adapter/pull/189). Unpublish single version in Npm.
This [issue](https://github.com/artipie/helm-adapter/issues/96) was also resolved but I'm not sure if it should be mentioned.
This [changes](https://github.com/artipie/artipie/issues/851) although refer to build lifecycle. This [page](https://github.com/artipie/artipie/wiki/Build-lifecycle) was added to clarify used tools

File: _posts/2021/02/2021-01-28-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- `FileStorage` `delete` operation was improved: now we also remove [empty directories](https://github.com/artipie/asto/issues/302) 
+recursively along with the removed file. This change fixed connected 
+[bug](https://github.com/artipie/docker-adapter/issues/435) in Docker-adapter
+- Basic authentication was [added](https://github.com/artipie/composer-adapter/issues/71) to Composer-adapter
+- Writing big files operation performance was [improved](https://github.com/artipie/asto/pull/304) in Asto
+- Npm-adapter now [supports](https://github.com/artipie/npm-adapter/issues/188) `tgz` upload, 
+to publish the package simple `curl` call can be used: `curl PUT your-archive.tgz`
+
+#### Granular Permissions For Docker Registry
+Docker-adapter support granular permissions, which means that some operations can be granted for
```
Comment (genryxy) [inline _posts/2021/02/2021-01-28-release-notes.md]: @olenagerasimova should be `supports`

File: _posts/2021/02/2021-01-28-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      - repository:some-image:push
+    mike:
+      - repository:some-image:push
+      - repository:some-image:pull
+    bob:
+      - registry:base:*
+      - registry:catalog:*
+```
+Each permission consists of three parts: `scope:name:action`, `scope` can be either "registry" for 
+the registry level operations or "repository". `Name` is the access recourse name, `action` can be
```
Comment (genryxy) [inline _posts/2021/02/2021-01-28-release-notes.md]: @olenagerasimova typo in the word `resource`

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, see two minor comments

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Comment by olenagerasimova** [comment]
@genryxy also thanks for the additions, I've added about npm unpublish, other changes seem too internal to be mentioned.

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**